lang/en.json

```diff
@@ -1,4 +1,51 @@
 {
+	"CYPHERSYSTEM.Init": "Initializing Cypher System",
```

Comment:
`"CYPHERSYSTEM.Init": "Initializing Cypher System"` and `"CYPHERSYSTEM.CanvasRendered": "The canvas was just rendered for scene:"` are system notifications that are only rendered in the console. Is it useful to have those translated?

Reply:
Maybe not, I just made sure I put a translation string everywhere I saw one.

Reply:
Deleted, can be put back if needed